2-run-automation-test.md

```diff
@@ -0,0 +1,131 @@
+# Running Automation test with Kobiton
+## 1. Prepare Kobiton configuration for excuting automation testing
+### 1.1 Get Kobiton Username and API Key
++ Go to  https://portal.kobiton.com
+
+**a. Get Kobiton Username**
+  + In the upper right hand corner, click on User icon and in the drop down menu, select 'Profile' 
+
+![alt text](./assets/click_profile.png )
+
++ You should see your username
+
+![alt text](./assets/username.png )
+
+**b.Get Kobiton API key**
++ Click on user icon in the upper right hand corner again and select 'Setting'
+
+![alt text](./assets/click_setting.png )
+
++ You will see your API key under the 'API Keys' 
+
+![alt text](./assets/api_key.png )
+
+1.2 Configure in your project TeamCity
++ In your project in TeamCity, on the left side on the page, click on 'Build Step' to set build steps
++ On ther right side on the page, Choose 'Command Line ' on the combo box 'Runner type'
++ Add command line to box. 
+
+Example:
+```
+  cd javascript
+  npm install
+  npm run android-web-test
+```
+
+![alt text](./assets/cmd.png )
+
+## 2. Write the automation test script
+2.1 Desired cap
++ In the navigation bar at the top of the Kobiton website, select Devices.
++ Hover over any device and click on the Automation settings button then a dialog appear in screen.
```

Comment:
"button then a dialog appear in screen." -> "button. A dialog box will then appear on the screen."